index.html

```diff
@@ -0,0 +1,126 @@
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <button>See project</button>
+            </li>
+            <li class="container2">
+                    <div class="works-pic"><img src="Assets/files/Snapshoot4.png" alt=""></div>
+                    <h2 class="works-heading">Multi-Post Stories</h2>
+                    <div class="canopy"><img src="Assets/files/Frame 2.png" alt=""></div>
+                    <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                <button>See project</button>
+            </li>
+        </ul>
+    </section>
+    <section id="about-self">
+        <h3 class="aboutself-heading">About myself</h3>
+        <p class="aboutself-paragragh">Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see  and have a project you need coded, don't hesitate to contact me.</p>
+        <h4 class="connect2">LET'S CONNECT</h4>
+            <ul id="social-icons">
+                <li><a href="#" class="icon"><img src="Assets/files/assets/github.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Linkedin icon.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Vector.jpg" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/twitter.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/medium.png" alt="Loading"></a></li>
+            </ul>
+        <button class="resume">Get my resume</button>
+
+        <li class="languages1">
+            <h3>Languages</h3>
+            <div class="arrow"><img src="Assets/files/arrow.png" alt=""></div>
+            <ul id="tech">
+                <li><button class="tag tag1"> Javascript</button></li>
+                <li><button class="tag tag2"> HTML</button></li>
+                <li><button class="tag tag3"> CSS</button></li>    
+            </ul>
+        </li>
+              <li class="works">
+                <label class="frameworks">Frameworks</label>
+                <img class="arrow1" src="Assets/files/DisabledArrow.png">
+                <div class="divider"></div>
+              </li>
+              
+              <li class="works">
+                <label class="skills">Skills</label>
+                <img class="arrow2" src="Assets/files/DisabledArrow.png">
+                <div class="divider"></div> <br> <br> <br> <br> <br> <br>
+              </li>
+        </ul> 
```

Comment:
After implementing the requested changes, I noticed that you have removed the opening tag for the `</ul>` tag on line `123` 👀

Now, by inspection, it seems the `li's` are not wrapped inside a `ul` tag, which is semantically wrong.

![image](https://github.com/denigit5/Portfolio/assets/78251927/52415061-4e4d-4b12-8434-2b2f83342d3e)

Please kindly adjust your markup 🤓👍